src/main/java/dan200/computercraft/ComputerCommand.java

```diff
@@ -0,0 +1,50 @@
+package dan200.computercraft;
+
+import dan200.computercraft.shared.computer.core.ComputerFamily;
+import dan200.computercraft.shared.computer.core.ServerComputer;
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import org.apache.commons.lang3.ArrayUtils;
+
+public class ComputerCommand extends CommandBase {
+
+    @Override
+    public String getName() {
+        return "computer";
+    }
+
+    @Override
+    public String getUsage(ICommandSender iCommandSender) {
+        return "computer <id> <value1> [value2]...";
+    }
+
+    @Override
+    public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
+        if( args.length < 2 ){
+            throw new CommandException( "Usage: /computer <id> <value1> [value2]..." );
+        }
+        int id;
+        try {
+            id = Integer.valueOf(args[0]);
+        }catch( NumberFormatException e ){
+            throw new CommandException( "Invalid ID" );
+        }
+        boolean found_valid_computer = false;
+        for( ServerComputer computer : ComputerCraft.serverComputerRegistry.getComputers() ){
+            if( computer.getID() == id && computer.getFamily() == ComputerFamily.Command ){
+                computer.queueEvent("computer_command", ArrayUtils.remove(args, 0));
+                found_valid_computer = true;
```

Comment:
There shouldn't be any computers with duplicate ids, so add a `break` here? Or, since we now have Java 8, if we make that assumption, this can be a whole lot nicer:
```Java
Optional<ServerComputer> optServerComp = ComputerCraft.serverComputerRegistry.getComputers().stream()
    .filter(c -> c.getID() == id)
    .filter(c -> c.getFamily() == ComputerFamily.Command)
    .findFirst();

if(optServerComp.isPresent())
{
    optServerComp.get().queueEvent("computer_command", ArrayUtils.remove(args, 0));
}
else
{
    throw new CommandException( "Computer #" + args[0] + " is not a Command Computer" );
}
```

Reply:
Did you read *anything* in the comments and reviews?  I previously supported 1 computer instance, and now support more - at SquidDev's request.  It is definitely possible to have more than 1 computer with the same ID.

Reply:
In fairness, that review is now hidden as you've made changes. Sometimes GH's features are counter-productive :/. 

@Lignum For reference, this was requested because Taken spawns pocket computers with the same ID as a command computer, meaning in its original form the command failed.

Reply:
Hmm, alright, in that case a `Stream` won't be any nicer than what we have now. Nevermind then.

Reply:
It could still be done with a `Stream` somewhat nicely:

```
ComputerCraft.serverComputerRegistry.getComputers().stream()
    .filter(c -> c.getID == id)
    .filter(c -> c.getFamily() == ComputerFamily.Command)
    .peek(c -> c.queueEvent("computer_command", ArrayUtils.remove(args, 0))
    .findFirst().orElseThrow(() -> new CommandException("No command computer with ID " + args[0] + " found");
```